### PR TITLE
PHPStan level 9

### PR DESCRIPTION
**Pull request by herndlm**
Not entirely sure if and where `FixWpStubs` is used though. Is it still needed?

### CODE REVIEW

File: src/Composer/FixWpStubs.php [hunk 1]
```diff
@@ -25,6 +25,11 @@ public static function php73Polyfill(Event $event): int
         $io->write('Removing duplicate is_countable() ...');
 
         $vendorDir = $event->getComposer()->getConfig()->get('vendor-dir');
+        if (!is_string($vendorDir) || strlen($vendorDir) <= 0) {
```
Comment (szepeviktor) [inline src/Composer/FixWpStubs.php:28]: Oh! 😱 negative-length strings :)

```suggestion
        if (!is_string($vendorDir) || strlen($vendorDir) !== '') {
```

Comment (herndlm) [reply src/Composer/FixWpStubs.php]: Those are the best strings. If you haven't worked with them, you haven't worked with strings!

Comment (szepeviktor) [reply src/Composer/FixWpStubs.php]: I have a special relations with strings.
https://gist.github.com/szepeviktor/6003fb6e2a35c0e5f9eb29c96623e4b7

**Comment by szepeviktor** [comment]
Thank you for your work.

**Comment by szepeviktor** [comment]
> Not entirely sure if and where `FixWpStubs` is used though. Is it still needed?

No, it is not needed.
This fix was for the pre-`php-stubs` era.

File: src/Composer/FixWpStubs.php (hunk 1 above)
Comment (herndlm) [reply src/Composer/FixWpStubs.php]: I'm a bit scared now.

**Pull request merged**